src/modules/UI/scenes/TransactionList/style.js

```diff
@@ -65,15 +65,33 @@ export const styles = {
   },
 
   // end of searchbar stuff
-
+  touchablBalanceBox: {
+    height: 200
+  },
   currentBalanceBox: {
     flex: 1,
     justifyContent: 'center'
   },
+  totalBalanceWrap: {
+    flex: 3,
+    alignItems: 'center',
+    backgroundColor: THEME.COLORS.TRANSPARENT
+  },
+  hiddenBalanceBoxDollarsWrap: {
```

Comment:
what does this do?

Reply:
It is a wrap around certain elements in the TxList Balance Box.